### PR TITLE
Fix: Added Confirmation Dialog in Passcode Activity

### LINKED ISSUE #86: Bug: Add a confirmation dialog in forgot passcode
**Summary:**
When user clicks on "Forgot passcode, login manually", he is directed to login page directly. However a cofirmation dialog should confirm whether he/she really wants to login manually. This will enhance the User Experience.

**Steps to reproduce:**
1. Go to Passcode Activity
2. Click on "Forgot passcode, login manually" button
3. You will be directed to the Login Activity

**Expected behaviour:**
Before directing to the login activity, a dialog should pop up to confirm the action.
**Observed behaviour:**
.No Confirmation takes place

**Device and Android version:**

Device - Mi A2 
Android 9 Pie

GIF:
ezgif com-video-to-gif (6)

### PR DESCRIPTION
**Pull request by rachittt**
Fixes #86 

GIF: 
ezgif com-video-to-gif (9)

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by rachittt** [comment]
@abhijitramesh can you please review this ?

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/passcode/PasscodeActivity.kt [hunk 1]
```diff
@@ -22,10 +25,21 @@ class PasscodeActivity : MifosPassCodeActivity(){
...
+        MaterialDialog.Builder().init(this)
+                .setMessage(R.string.login_using_password_confirmation)
+                .setPositiveButton(getString(R.string.logout),
+                        DialogInterface.OnClickListener { dialog, which ->
+                            val intent = Intent(this, LoginActivity::class.java)
+                            intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
+                            startActivity(intent)
+                            finish()
+                            //Show successful logout Toast.
+                            Toast.makeText(applicationContext, "Logged Out Successfully", Toast.LENGTH_SHORT).show()
```
Comment (luckyman20) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/passcode/PasscodeActivity.kt]: Use Snackbar.

**Comment by luckyman20** [comment]
@abhijitramesh Can you please merge this?

**Comment by rachittt** [comment]
> Please change the dialog message to the one you've added in the `strings.xml` file

@harshshinde07 The GIF is not updated but I've put that string resource as the dialog message only 😃

**Pull request merged**